data-sets-tests/src/test/java/org/zowe/unix/files/tests/UnixFilesDeleteFileTest.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2019
+ */
+package org.zowe.unix.files.tests;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+
+import org.apache.http.HttpStatus;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.zowe.api.common.errors.ApiError;
+import org.zowe.tests.AbstractHttpIntegrationTest;
+import org.zowe.unix.files.exceptions.FileNotFoundException;
+import org.zowe.unix.files.exceptions.NotAnEmptyDirectoryException;
+import org.zowe.unix.files.exceptions.UnauthorisedFileException;
+import org.zowe.unix.files.model.UnixDirectoryAttributesWithChildren;
+import org.zowe.unix.files.model.UnixDirectoryChild;
+import org.zowe.unix.files.model.UnixEntityType;
+
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+
+
+public class UnixFilesDeleteFileTest extends AbstractHttpIntegrationTest {
+
+    static final String UNIX_FILES_ENDPOINT = "unixfiles";
+    static final String TEST_DIRECTORY = System.getProperty("server.test.directory");
+
+    @BeforeClass
+    public static void setUpEndpoint() throws Exception {
+        RestAssured.basePath = UNIX_FILES_ENDPOINT;
+    }
+
+    @Test
+    public void testDeleteUnixFileContent() throws Exception {
+        final String fvtDeleteFile = TEST_DIRECTORY + "/deleteTestDirectoryAccess/deleteFileWithWritePermission";
+
+        RestAssured.given().when().delete(fvtDeleteFile).then().statusCode(HttpStatus.SC_NO_CONTENT);
+        
+        ApiError expectedError = new FileNotFoundException(fvtDeleteFile).getApiError();
+        
+        RestAssured.given().when().get(fvtDeleteFile)
+            .then().statusCode(HttpStatus.SC_NOT_FOUND).contentType(ContentType.JSON)
+            .body("message", equalTo(expectedError.getMessage()));
+    }
+
+    @Test
+    public void testDeleteUnixFilePermissionDenied() throws Exception {
+        final String noPermissionDenied = TEST_DIRECTORY
+                + "/deleteTestDirectoryWithoutAccess/deleteFileWithoutWritePermission";
+        ApiError expectedError = new UnauthorisedFileException(noPermissionDenied).getApiError();
+
+        RestAssured.given().when().delete(noPermissionDenied).then().statusCode(HttpStatus.SC_FORBIDDEN).body("message",
```

Comment:
Change the line breaking of the call chain to have the assertion functions on new lines, similar to how you've done it for the previous tests. So when the call to then() is made switch to a new line for readability and consistency

Reply:
formatted